src/silx/gui/widgets/RangeSlider.py

```diff
@@ -122,7 +122,12 @@ def __init__(self, parent=None):
     def event(self, event):
         t = event.type()
         if t == qt.QEvent.HoverEnter or t == qt.QEvent.HoverLeave or t == qt.QEvent.HoverMove:
-            return self.__updateHoverControl(event.pos())
+            if hasattr(event, "position"):
+               # qt-6
+               return self.__updateHoverControl(event.position().toPoint())
+            else:
+               # qt-5
+               return self.__updateHoverControl(event.pos())
```

Comment:
```suggestion
            return self.__updateHoverControl(
                self._mouseEventPosition(event)
            )
```

Reply:
No, it is a different type of event and it has to be handled differently. One is a mouseEvent and the other a HoverEvent.

Reply:
indeed you are right, I was thinking that the binding should also be used here